packages/scxa-faceted-search-results-v5/src/FilterSidebar.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import CheckboxFacetGroup from './facetgroups/CheckboxFacetGroup'
+import {withFetchLoader} from "@ebi-gene-expression-group/atlas-react-fetch-loader";
+import MultiselectDropdownFacetGroup from "./facetgroups/MultiselectDropdownFacetGroup";
+
+const CheckboxFacetGroupWithFetchLoader = withFetchLoader(CheckboxFacetGroup)
+const MultiselectDropdownFacetGroupWithFetchLoader = withFetchLoader(MultiselectDropdownFacetGroup)
+
+class FilterSidebar extends React.Component {
```

Comment:
It doesn’t seem that `FilterSidebar` is stateful. We should get rid of the state and rewrite it as a function to make that explicit.